src/dispatchers/touchDispatcher.ts

```diff
@@ -0,0 +1,138 @@
+///<reference path="../reference.ts" />
+
+module Plottable {
+export module Dispatcher {
+  export type TouchCallback = (p: Point) => any;
+
+  export class Touch extends AbstractDispatcher {
+    private static _DISPATCHER_KEY = "__Plottable_Dispatcher_Touch";
+    private translator: _Util.ClientToSVGTranslator;
+    private _lastTouchPosition: Point;
+    private _moveBroadcaster: Core.Broadcaster<Dispatcher.Touch>;
+    private _startBroadcaster: Core.Broadcaster<Dispatcher.Touch>;
+    private _endBroadcaster: Core.Broadcaster<Dispatcher.Touch>;
+    private _processMoveCallback: (e: TouchEvent) => any;
+    private _processStartCallback: (e: TouchEvent) => any;
+    private _processEndCallback: (e: TouchEvent) => any;
+
+    /**
+     * Get a Dispatcher.Touch for the <svg> containing elem. If one already exists
+     * on that <svg>, it will be returned; otherwise, a new one will be created.
+     *
+     * @param {SVGElement} elem A svg DOM element.
+     * @return {Dispatcher.Touch} A Dispatcher.Touch
+     */
+    public static getDispatcher(elem: SVGElement): Dispatcher.Touch {
+      var svg = _Util.DOM.getBoundingSVG(elem);
+
+      var dispatcher: Touch = (<any> svg)[Touch._DISPATCHER_KEY];
+      if (dispatcher == null) {
+        dispatcher = new Touch(svg);
+        (<any> svg)[Touch._DISPATCHER_KEY] = dispatcher;
+      }
+      return dispatcher;
+    }
+
+    /**
+     * Creates a Dispatcher.Touch.
+     * This constructor not be invoked directly under most circumstances.
+     *
+     * @param {SVGElement} svg The root <svg> element to attach to.
+     */
+    constructor(svg: SVGElement) {
+      super();
+
+      this.translator = _Util.ClientToSVGTranslator.getTranslator(svg);
+
+      this._lastTouchPosition = { x: -1, y: -1 };
+      this._moveBroadcaster = new Core.Broadcaster(this);
+      this._processMoveCallback = (e: TouchEvent) => this._measureAndBroadcast(e, this._moveBroadcaster);
+      this._event2Callback["touchmove"] = this._processMoveCallback;
+
+      this._startBroadcaster = new Core.Broadcaster(this);
+      this._processStartCallback = (e: TouchEvent) => this._measureAndBroadcast(e, this._startBroadcaster);
+      this._event2Callback["touchstart"] = this._processStartCallback;
+
+      this._endBroadcaster = new Core.Broadcaster(this);
+      this._processEndCallback = (e: TouchEvent) => this._measureAndBroadcast(e, this._endBroadcaster);
+      this._event2Callback["touchend"] = this._processEndCallback;
+
+      this._broadcasters = [this._moveBroadcaster, this._startBroadcaster, this._endBroadcaster];
+    }
+
+    protected _getWrappedCallback(callback: Function): Core.BroadcasterCallback<Dispatcher.Touch> {
+      return () => callback(this.getLastTouchPosition());
```

Comment:
Why don't we follow the consistency of Mouse and have the getLastTouchPosition be handed as an arg